ES/ESdd009.xml

```diff
@@ -178,118 +278,144 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                                 monastery. Subsequently, the donor <persName ref="PRS14596WalattaSellase">Walatta Śǝllāse</persName> and her sister Walatta
                               Krǝstos are mentioned. </note>
                         </item>
-                        <item xml:id="e1">- The title of the work, ድርሳነ ማህያዊ, and the internal
-                           number of the monastic library (802) are written on <locus target="#1r">1r</locus>. &#xD; - A space for a name of the supplicant(s) is left
-                           unfilled: <locus target="#105va">105va</locus>.&#xD; - Erasures marked
-                           with two parallel lines: <locus target="#25rb">25rb</locus>, 47ra.&#xD; -
-                           Pen trials: <locus target="#78r">78r</locus>, 91r.</item>
-                        <item xml:id="e2"><!--2019-04-02+02:00 Updated DD-009 
-            from DomLib -->
-                           <desc type="findingAid">- Daily readings of Text I are indicated by the
+                        
+                        <item xml:id="e1">
+                           <locus target="#1r"/>
+                           <desc type="Unclear">The title of the work<title ref="LIT1291Dersan"/> ድርሳነ ማህያዊ is written in large letters the middle of the folio; the internal
+                              number of the monastic library (802) is written in the lower part of the folio. </desc> 
+                           </item>
+                           <item xml:id="e2">A space for a name of the supplicant(s) is left
+                           unfilled: <locus target="#105va"/>.
+                           </item>
+                        <item  xml:id="e3">Erasures marked
+                           with two parallel lines: <locus target="#25rb #47ra"></locus>.
+                        </item>
+                        
+                           <item xml:id="e4">
+                              Pen trials: <locus target="#78r #91r"/>.  
+                           </item>
+      
+                        <item xml:id="e5">
+                           <desc type="findingAid">Daily readings of <ref target="ms_i1"/> Text I are indicated by the
                               names of the praying hours and the days of the week written in the
-                              upper margin, in the main hand: zä-nǝwam zä-sänuy: <locus target="#3ra">fol. 3ra</locus> za- mänfäqä lelit zä-śälus: <locus target="#14ra">fol. 14ra</locus> zä-nägh zä-räbuʿ: <locus target="#23rb">fol. 23rb</locus> zä-3 säʿat zä-ḥamus: <locus target="#31rb">fol. 31rb</locus> zä-6 säʿat zä-ʾarb: <locus target="#42va">fol. 42va</locus> zä-9 säʿat zä-qädam: <locus target="#53vb">fol. 53vb</locus> zä-śärk zä-ʾǝḥud: <locus target="#62rb">fol. 62rb</locus>. - Daily readings of Text II are
+                              upper margin, in the main hand: za-nǝwām za-sanuy: <locus
+                                 target="#3ra"/> za-manfaqa lelit za-śalus: <locus
```

Comment:
Please make these notes  more transparent. You need to encode transcribed passage like this: <foreign xml:lang="gez">za-nǝwām za-sanuy</foreign> on <locus target="#3ra"/>; .....

Reply:
ok